android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -510,39 +515,18 @@ public void disconnect() {
 
     // ===== SEND STRING ON DATA TRACK ======================================================================
     public void sendString(String message) {
-          if (localDataTrack != null) {
-                localDataTrack.send(message);
-          }
+        if (localDataTrack != null) {
+            localDataTrack.send(message);
         }
+    }
 
     // ===== BUTTON LISTENERS ======================================================================
     private static void setThumbnailMirror() {
         if (cameraCapturer != null) {
             CameraCapturer.CameraSource cameraSource = cameraCapturer.getCameraSource();
             final boolean isBackCamera = (cameraSource == CameraCapturer.CameraSource.BACK_CAMERA);
-
-            // Previously, only thumbnailView has its mirroring set.
```

Comment:
As it's still possible for our app to render multiple local video, we still need to `setMirror`ing to all local video renderers.